### ISSUE TITLE
[Nuxt SSR]: toRefs() expects a reactive object but received a plain one.

### ISSUE THREAD

**Issue by AlexisWalravens** [Kong/swrv#89]
Hi,

I get a `toRefs() expects a reactive object but received a plain one.` on every useSWR I use on a page in Nuxt.
The error occurs only server side:
![image](https://user-images.githubusercontent.com/1764824/94899967-84865880-0494-11eb-8f29-d2cac96981bf.png)

Everything still works as it is only a warn but would be nice if we could get rid of the error.

**Comment by darrenjennings** [comment]
@AlexisWalravens thanks for the note, would you be open to submitting a PR?

**Comment by AlexisWalravens** [comment]
@darrenjennings I really don't know where it comes from.
The only toRefs in the projet is in the return of use-swrv.ts
```
  return {
    ...toRefs(stateRef),
    mutate: revalidate
  } as IResponse<Data, Error>
```

And I tried to log an `isReactive(stateRef)` juste before that return but it returns `true` twice.

**Comment by AlexisWalravens** [comment]
Solved in 0.7.7.
